LDM-592.tex

```diff
@@ -967,6 +967,18 @@ \section{Data Release Processing}
 
 }
 
+\usecase{DRP32}{Use of rerun collections}{%
+
+\item
+A Developer runs some processing starting from outputs produced by another Developer, where those outputs are specified as inputs using a label given as a single string (possibly with path-like namespace components).
```

Comment:
I'd like to see the system use reruns too, so any processing gets a "rerun" tag (e.g. 2025/DRP/RC2).  Then I can look at those outputs or reprocess part of that data in some different way.

I.e. Everyone will say `butler = Butler(URI, rerun="2025/DRP/RC2")` in their notebooks.  Anyone wanting to reprocess data will say something like `--rerun 2025/DRP/RC2:rhl/tmp` and then `butler = Butler(URI, rerun="rhl/tmp")` to look at the results.

Reruns *supersede* explicit specification of inputs and outputs (except possibly deep in processing scripts that no-one outside NCSA ever sees)

Reply:
I think the difference between what's in the text and in @RobertLuptonTheGood's comment is policy; it's whether we declare that this is the *only* way processing runs are labeled.

I think that's reasonable (and it will actually be fairly strongly baked into the Gen3 design), but I'll defer to @timj as to whether it belongs in a use case.